api/support/verify_state.py

```
@@ -0,0 +1,294 @@
+    """Run one verifier round for a chain. Returns Influx-format lines to emit.
+
+    Always emits at least one ``verifier_status`` line so dashboards can tell
+    "verifier round happened" from "no sample" — never returns an empty list.
+    """
+    ts_ns = time.time_ns()
+    chain_lower = chain.lower()
+    chain_state = state_data.get(chain_lower)
+    if not chain_state or not chain_state.get("old_block"):
+        # Missing OLD_BLOCK — could be update_state outage, blob read failure, etc.
+        # Treat as anchor unavailable so the failure is visible, not silent.
+        logging.warning(f"verify_state: no old_block for {chain}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)